.gitignore

```
@@ -1,2 +1,6 @@
 *.egg-info
 *.pyc
+*.idea
+
+#default project path
+**/python-template
```

Just need this I think?
```suggestion
python-template
```